client/ayon_maya/plugins/publish/extract_look.py

```diff
@@ -513,7 +514,7 @@ def process(self, instance):
                 with no_workspace_dir():
                     with lib.attribute_values(remap):
                         with lib.maintained_selection():
-                            cmds.select(sets, noExpand=True)
+                            cmds.select(sets + texture_objs, noExpand=True)
```

Comment:
Beware: This will INCLUDE all construction history and inputs from the Texture Reference objects. If they are animated, the animation will come along. If they are constrained by something, those inputs will come along, etc.

Something to be aware of ;) not critical for now as we can explain for now to not do crazy things. But note that e.g. also the _parent hierarchy_ of the texture reference objects will come along.